internal/entity/statistic_task_entity.go

```diff
@@ -0,0 +1,30 @@
+package entity
+
+import "fmt"
+
+type TaskStatistic struct {
+	Type     string `json:"type"`
+	Status   string `json:"status"`
+	Finished string `json:"finished"`
+	Spent    string `json:"spent"`
+}
+
+const (
+	TaskStatusCreated    = "created"
+	TaskStatusInProgress = "in_progress"
+	TaskStatusFinished   = "finished"
+)
+
+const TaskIDPrefix = "task_id_"
+
+func TaskIDFromString(id string) string {
```

Comment:
Why not to move this function to a separate package, so it will be `converter.TaskIDFromString`?